lib/page/SettingsPage.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:project_apraxia/widget/form/InvalidateWaiverForm.dart';
+
+import 'package:project_apraxia/widget/form/UpdateUserForm.dart';
+
+class SettingsPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 3,
+      child: Scaffold(
+        appBar: AppBar(
+          title: Text("Settings"),
+          bottom: TabBar(
+            tabs: <Widget>[
+              Text("Account"),
+              Text("Prompts"),
+              Text("Waivers")
```

Comment:
Could you wrap these in Tab widgets?
Tab(child: Text("String"))

I was going to add it but then I was worried that we would need another reviewer